easybuild/easyblocks/v/vmd.py

```diff
@@ -148,7 +148,11 @@ def configure_step(self):
         # PTHREADS: enable support for POSIX threads
         # COLVARS: enable support for collective variables (related to NAMD/LAMMPS)
         # NOSILENT: verbose build command
-        self.cfg.update('configopts', "LINUXAMD64 LP64 IMD PTHREADS COLVARS NOSILENT", allow_duplicate=False)
+        # FLTK: enable the standard FLTK GUI
+        # TK: enable TK to support extension GUI elements
+        # OPENGL: enable OpenGL
+        self.cfg.update(
+            'configopts', "LINUXAMD64 LP64 IMD PTHREADS COLVARS NOSILENT FLTK TK OPENGL", allow_duplicate=False)
```

Comment:
Adding FTLK & co should be done conditionally, only if those dependencies are available?

There may be VMD easyconfigs out there that don't have those deps included.

You can use the `get_software_root` function to check whether a dep is provided, there are several examples of that in other easyblocks.

Let us know if you need help with that...

Reply:
@boegel they are actually required (see earlier in the file, errors if not included)

Reply:
@boegel they are actually required (see earlier in the file, errors if not included)